scripts/nlp_language_modeling/prepare_packed_ft_dataset.py

```diff
@@ -83,9 +88,15 @@ def tokenize_dataset(cfg: 'DictConfig'):
     # using the same template as SFT/PEFT script. This may be overkill but guarantees the preprocess settings
     # are identical to normal SFT training
     data_cfg = cfg.model.data.train_ds
+    if os.path.isdir(cfg.tokenizer_path):
+        # pass in a Hugging Face folder which contains tokenizer.json
+        tokenizer = get_nmt_tokenizer(library="huggingface", model_name=cfg.tokenizer_path)
```

Comment:
for hf tokenizer, there is another args `use_fast` which determines the huggingface implementation of the tokenizer even loading the same file. we might want to add this option as well ?